packages/paste-website/src/pages/patterns/copy-to-clipboard/index.mdx

```diff
@@ -0,0 +1,454 @@
+---
+title: Copy to clipboard
+description: Give customers guidance on what steps to take if content is either unavailable or can't be displayed.
+slug: /patterns/copy-to-clipboard/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Card} from '@twilio-paste/card';
+import {Grid, Column} from '@twilio-paste/grid';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Table, Tr, Th, Td, THead, TBody} from '@twilio-paste/table';
+import {Text} from '@twilio-paste/text';
+import {useUID} from 'react-uid';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/patterns/copy-to-clipboard/"}}) {
```

Comment:
Let's use the package name here, without the `@twilio-paste/` prefix.  So `eq: "/libraries/clipboard-copy-library"`

Reply:
same comment as above